builders/tools/suitesparse.py

```diff
@@ -35,7 +35,7 @@ def install(dir_name,version=None):
                 f.write('LAPACK = -L'+os.path.join(dir_name,'lib')+' -lopenblas\n')
             finally:
                 f.close()
-            if subprocess.call(['make'],cwd=suitesparse_dir):
+            if subprocess.call(['make','library'],cwd=suitesparse_dir):

```

Comment:
`make` also runs tests, `make library` doesn't. I think `make library` ran into a linking problem with `gcc_s` required by gfortran and failed. This was the easiest way to fix it.